src/docker-osx-dev

```diff
@@ -600,10 +600,19 @@ function check_prerequisites {
     log_error "This script should only be run on OS X"
     exit 1
   fi
+}
 
+#
+# Update Brew formulae and HomeBrew
+#
+# Non-breaking. Falls through with warning if fails.
+#
+function brew_update {
+  log_info "Updating HomeBrew"
   if ! type brew > /dev/null 2>&1 ; then
-    log_error "This script requires HomeBrew, but it's not installed. Aborting."
-    exit 1
+    log_warn "HomeBrew not found, attempting to continue without"
```

Comment:
Just for clarity, may want to have the comment say "HomeBrew not found, will not attempt to update it".